wolfcrypt/src/tfm.c

```diff
@@ -3295,9 +3295,21 @@ int fp_exptmod_nct(fp_int * G, fp_int * X, fp_int * P, fp_int * Y)
 #if defined(WOLFSSL_ESP32_CRYPT_RSA_PRI_EXPTMOD)
    int retHW = FP_OKAY;
 #endif
+    if (P == NULL) {
```

Comment:
This file has a historically different 3 space Indentation. Please have it match existing and do not refactor anything. 

Was this NULL check really needed? This should never be NULL based on calling functions. Although technically this function isn't even set as WOLFSSL_LOCAL. If you need this added its fine. 

Also I recommend adding include for logging.h at top below wolfmath.h.

Reply:
Agree on convention; I missed the 3-space, thanks for catching that. 

I removed the NULL check, as I did not encounter a need for it. I was only concerned about this expansion:

```
#define fp_iszero(a) (((a)->used == 0) ? FP_YES : FP_NO)
```

Not an issue considering how little this file changes, re: calling functions.

Also added:  `#include <wolfssl/wolfcrypt/logging.h>`